packages/salesforcedx-vscode-core/src/commands/forceSourceStatus.ts

```diff
@@ -55,9 +77,9 @@ const parameterGatherer = new EmptyParametersGatherer();
 export async function forceSourceStatus(
   this: FlagParameter<SourceStatusFlags>
 ) {
-  // tslint:disable-next-line:no-invalid-this
-  const flag = this ? this.flag : undefined;
-  const executor = new ForceSourceStatusExecutor(flag);
```

Comment:
nit: knowing what we know now would it make sense to add a type guard here for this? 
```
if (!this instanceof FlagParameter) { throw error} 
```
I don't even know if this works with `this` so marking this a nit